sites/footers/has-footer-contextual-en.html

```diff
@@ -0,0 +1,92 @@
+---
+{
+	"title": "Show contextual links in footer",
+	"language": "en",
+	"altLangPage": "no-footer-contextual-fr.html",
+	"secondlevel": false,
+	"dateModified": "2022-06-20",
+	"share": "true",
+	"hasFooterContextual": "true"
+}
+---
+<p>By setting the <code>hasFooterContextual</code> variable to "true", all utility links from the contextual footer section will be visible on page load.</p>
+
+<ul>
+	<li>The contextual section must contain:
+		<ul>
+			<li>a meaningful title.</li>
+			<li>a minimum of one (1) link to a maximum of six (6) links (ideal and default presentation is three (3) links).</li>
+		</ul>
+	</li>
+</ul>
+
+<p>The contextual section is defined in <code>_config.yml</code>.</p>
+
+<h2>Expected output code</h2>
+<pre><code>
+&lt;footer id="wb-info"&gt;
```

Comment:
Remove the empty line in the code block

```suggestion
<pre><code>&lt;footer id="wb-info"&gt;
```